### DISCUSSION TITLE
Centering CTkTopLevel on its parent

### DISCUSSION THREAD

**Discussion by cristian64** [TomSchimansky/CustomTkinter#1621] [Q&A] [answered]
Hi!

I've noticed that `CTkTopLevel` windows are not centered on their parent window upon creation. How expected is this? Can this be considered a bug, or is this perhaps a Tkinter limitation? 

(Of course `CTkInputDialog` is equally affected.)

As a related question, what would be a reasonable workaround? Center window manually with some delay so that the geometry of the child can be known?

Thanks!

**Comment by ElectricCandlelight** [comment, marked as answer]
It's the default behaviour of Tkinter. 

I put in a workaround for this in CTkMessagebox

[CTkMessagebox](https://github.com/Akascape/CTkMessagebox/blob/main/CTkMessagebox/ctkmessagebox.py) 

Lines 54 to 58
